lib/pana.dart

```diff
@@ -79,8 +79,18 @@ class PackageAnalyzer {
 
     PubSummary summary;
     if (upgrade.exitCode == 0) {
-      summary = PubSummary.create(upgrade.stdout, path: pkgDir);
-      log.info("Package version: ${summary.pkgVersion}");
+      try {
+        summary = PubSummary.create(upgrade.stdout, path: pkgDir);
+      } catch (e, stack) {
+        log.severe("Problem summarizing package", e, stack);
+        //(TODO)kevmoo - should add a helper that handles logging exceptions
```

Comment:
I don't mind writing two lines of code in such cases, but I don't oppose the helper either.